app/src/main/res/values/strings.xml

```diff
@@ -38,4 +38,5 @@
     <string name="comment_url" translatable="false">https://codeforces.com/blog/entry/%1$d?#comment-%2$d</string>
     <string name="share_with_friends">Share with friends</string>
     <string name="shared_through_cw">Shared through Codeforces Watcher. Find it on Google Play.</string>
+    <string name="no_users">You don’t have any users to watch yet.\nAdd the first one using «+» at the bottom of the screen.</string>
```

Comment:
There is no `\n` in Russian and Ukranian versions. You probably need to add them.